src/token/erc6909/extensions/erc6909_content_uri.cairo

```diff
@@ -0,0 +1,62 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts for Cairo v0.14.0 (token/erc6909/extensions/erc6909_votes.cairo)
+
+use starknet::ContractAddress;
+
+/// # ERC6909ContentURI Component
+///
+/// The ERC6909ContentURI component allows to set the contract and token ID URIs.
+/// The internal function `initializer` should be used ideally in the constructor.
+#[starknet::component]
+pub mod ERC6909ContentURIComponent {
+    use openzeppelin::token::erc6909::ERC6909Component;
+    use openzeppelin::token::erc6909::interface;
+
+    #[storage]
+    struct Storage {
+        ERC6909ContentURI_contract_uri: ByteArray,
+    }
+
+    #[embeddable_as(ERC6909ContentURIImpl)]
+    impl ERC6909ContentURI<
+        TContractState,
+        +HasComponent<TContractState>,
+        +ERC6909Component::HasComponent<TContractState>,
+        +ERC6909Component::ERC6909HooksTrait<TContractState>,
+        +Drop<TContractState>
+    > of interface::IERC6909ContentURI<ComponentState<TContractState>> {
+        /// @notice The contract level URI.
+        /// @return The URI of the contract.
+        fn contract_uri(self: @ComponentState<TContractState>) -> ByteArray {
+            self.ERC6909ContentURI_contract_uri.read()
+        }
+
+        /// @notice Token level URI
+        /// @param id The id of the token.
+        /// @return The token level URI.
+        fn token_uri(self: @ComponentState<TContractState>, id: u256) -> ByteArray {
+            let contract_uri = self.contract_uri();
+            if contract_uri.len() == 0 {
+                return "";
+            } else {
+                return format!("{}{}", contract_uri, id);
+            }
```

Comment:
```suggestion
            if contract_uri.len() == 0 {
                ""
            } else {
                format!("{}{}", contract_uri, id)
            }
```

Early return syntax is not needed here

Reply:
Fixed in [40538f6](https://github.com/OpenZeppelin/cairo-contracts/pull/1040/commits/40538f672aaef69bf8dc96d3c9a86b3d4c9abbb2)